### ISSUE TITLE
Initial Focus Position

### ISSUE THREAD

**Issue by mgifford** [w3c/wcag-em-report-tool#534]
When evaluating the OpenACR editor I noticed that the focus didn't start at the top of the page, but rather with the main content.  I'd looked at this historically with Drupal's search.  What is the best experience for screen reader users when navigating a new page. In that we'd decided to start at the top of the page, rather than with the search box. Partly this was for consistency, but we understood this was the preferred pattern for screen reader users. So I opened a issue there:
  https://github.com/GSA/openacr/issues/332

But of course, this is also an issue in the WCAG-EM Tool, as we based it off of yours. So is this a bug in svelte? Is this an intentional design decision for the WCAG-EM tool? Is this a bug in Svelte which should be changed there? 

<img width="710" alt="Screen Shot of OpenACR with focus on the first element in the main content on load." src="https://user-images.githubusercontent.com/116832/153904598-9f35ac2b-05b3-4ef7-ba7f-da13c77bff78.png">
